system-tests/e2e-transfer-test/control-plane/build.gradle.kts

```
@@ -0,0 +1,21 @@
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":core"))
+    api(project(":data-protocols:ids"))
+    api(project(":extensions:filesystem:vault-fs"))
+    api(project(":extensions:http"))
+    api(project(":extensions:iam:iam-mock"))
+    api(project(":extensions:api:control"))
+    api(project(":extensions:in-memory:assetindex-memory"))
+    api(project(":extensions:in-memory:transfer-store-memory"))
+    api(project(":extensions:in-memory:negotiation-store-memory"))
+    api(project(":extensions:in-memory:contractdefinition-store-memory"))
+
+    api(project(":extensions:sync-data-transfer:sync-data-transfer-api"))
+    api(project(":extensions:sync-data-transfer:sync-data-transfer-consumer"))
+    api(project(":extensions:sync-data-transfer:sync-data-transfer-provider"))
+    api(project(":extensions:http-receiver"))
+}
```

Missing empty line at EOF
```suggestion
}

```